tests/test_minimizer.py

```diff
@@ -26,6 +27,26 @@ def test_minimizer():
     mol_a = all_mols[1]
     mol_b = all_mols[4]
 
+    for mols in [[mol_a], [mol_b], [mol_a, mol_b]]:
+        x_host = minimizer.minimize_host_4d(mols, host_config, ff)
+        assert x_host.shape == complex_coords.shape
+
+
+@pytest.mark.nightly(reason="Currently not used in practice")
```

Comment:
We may eventually write down functional requirements for pre-equilibration in timemachine, which might allow alternate implementations of this step to be considered for use in practice.

(Based on inferred requirements, current test coverage of `minimize_host_4d`, and initial investigation in https://github.com/proteneer/timemachine/pull/978 , the `equilibrate_host_barker` option implemented in timemachine appeared to offer a uniform improvement over the method currently used. But I am likely missing requirements.)

Reply:
Based on what spawned the original `minimize_host_4d` implementation, those two molecules caused failures due to clashes in the protein.

Not sure, there is much of a requirement beyond 'leads to a stable simulation'. From the barker tests it seems like it would be a fine drop-in replacement, would just need to test over a larger set of compounds to make sure there isn't some edge case.